Python/bltinmodule.c

```diff
@@ -2657,7 +2715,10 @@ PyDoc_STRVAR(zip_doc,
 The zip object yields n-length tuples, where n is the number of iterables\n\
```

Comment:
You should update the prototype: `zip(*iterables)` => `zip(*iterables, strict=False)`.